lib/storage/client.js

```diff
@@ -1796,6 +1796,65 @@ class Client extends CryptoConsumer {
         throw new Error('Error while updating subscription.')
     }
   }
+
+  async createRealmGroup(realmName, groupName) {
+
+    const payload = {
+      name:  groupName
+    }
+
+    let response = await this.authenticator.tsv1Fetch(
+      `${this.config.apiUrl}/v1/identity/realm/info/${realmName}/group`,
+      {
+        method: 'POST',
+        body: JSON.stringify(payload)
+      }
+    )
+
+    return validateResponseAsJSON(response)
+  }
+
+  async updateRealmGroup(realmName,tozId,tozStoreId){
+
+    const payload = {
+      attributes: {tozStoreId: [tozStoreId]}
```

Comment:
can we update this to pass in 
`async updateRealmGroup(realmName,groupID,attributes={})`

and to the method pass in the attributes object, since if this is used for anybody else they may not want this

Reply:
Okay Katie. I update the same